prov/gni/include/gnix_nic.h

```diff
@@ -194,6 +194,7 @@ struct gnix_nic {
 	gni_mem_handle_t irq_mem_hndl;
 	void *irq_mmap_addr;
 	size_t irq_mmap_len;
+	int requires_lock;
```

Comment:
It's a mistake. I'll address that.
-- Jim




On 4/14/16, 5:25 PM, "Sung-Eun Choi" <notifications@github.com> wrote:

>In 
>prov/gni/include/gnix_nic.h
><https://github.com/ofi-cray/libfabric-cray/pull/776#discussion_r59802215>
>:
>> @@ -194,6 +194,7 @@ struct gnix_nic {
>>  	gni_mem_handle_t irq_mem_hndl;
>>  	void *irq_mmap_addr;
>>  	size_t irq_mmap_len;
>> +	int requires_lock;
>Why is this one an integer and the rest bools?
>—
>You are receiving this because you authored the thread.
>Reply to this email directly or
>view it on GitHub 
><https://github.com/ofi-cray/libfabric-cray/pull/776/files/e88688645fe4700
>d5e9ac9c2697f4a36b7bd0ab8#r59802215>
>
>